src/App.tsx

```diff
@@ -26,26 +27,37 @@ function App() {
 
   return (
     <ThemeProvider theme={theme}>
-    <Router>
-      <div>
-        <Route component={SearchBar} />
-        <Switch>
-          <Route exact path="/suitcase" component={Suitcase} />
-          <Route path={"/suitcase/:topicId"}
-            render={props =>
-              <TopicViews topicId={props.match.params.topicId} />
-            } />
-          <Route exact path="/favorites" component={Favorites} />
-          <Route exact path="/guides" component={Guides} />
-          <Route path={"/guides/:lessonId"}
-            render={props =>
-              <PlaylistPage lessonId={props.match.params.lessonId} />
-            } />
-          <Route path="/troubleshoot" component={Troubleshoot} />
-        </Switch>
-      </div>
-      <TabBar />
-    </Router>
+      <Router>
+        <div style={{ display: "flex", flexDirection: "column" }}>
+          <div>
+            <Route component={SearchBar} />
+          </div>
+          <div>
+            <Switch>
+              <Route exact path="/suitcase" component={Suitcase} />
+              <Route path={"/suitcase/:topicId"}
+                render={props =>
+                  <TopicViews topicId={props.match.params.topicId} />
+                } />
+              <Route exact path="/favorites" component={Favorites} />
+              <Route exact path="/guides" component={Guides} />
+              <Route path={"/guides/:lessonId"}
+                render={props =>
+                  <PlaylistPage lessonId={props.match.params.lessonId} />
+                } />
+              <Route exact path="/troubleshoot" component={Troubleshoot} />
+              <Route path={"/Troubleshoot/:helpId"}
+                render={props =>
```

Comment:
NICE I think this routing approach is quite clean and extensible. Thanks for iterating on this with different versions!